libraries/chain/db_market.cpp

```diff
@@ -775,7 +786,10 @@ bool database::fill_limit_order( const limit_order_object& order, const asset& p
    const account_object& seller = order.seller(*this);
    const asset_object& recv_asset = receives.asset_id(*this);
 
-   auto issuer_fees = pay_market_fees( recv_asset, receives );
+   auto issuer_fees = ( head_block_time() < HARDFORK_1268_TIME ) ? 
```

Comment:
please also add comment here so we can just leave `pay_market_fees(seller, recv_asset, receives);` after hardfork. thanks.

Reply:
sorry, in this one is not possible to remove it, my mistake, please ignore.